src/lib.rs

```diff
@@ -52,6 +52,12 @@ pub struct Response {
     pub tag: String,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ErrorInformation {
+    pub r#type: String,
```

Comment:
While it's not something super-critical, I'm not entirely convincing of using a raw keyword here. My main fear is the possibility of it confusing some editors/IDEs. Would it possible to use serde's rename to set this field name to something else? Do we lose too much context information doing it that way?

Reply:
I renamed the field to `error_type` and serde serializes/deserializes it to `type`. Let me know if that's not what you meant here.

Reply:
That's great, thanks!